### DISCUSSION TITLE
Proper package installation & maintenance with UniGetUI & Chocolatey

### DISCUSSION THREAD

**Discussion by Infinitay** [Devolutions/UniGetUI#3035] [Q&A]
Before finding UniGetUI I was using Chocolatey as my primary package manager. I used Chocolatey to install frameworks/dev tools or applications. For the past few months I've been trying to transition into using WinGet instead via UnIGetUI. I just noticed today after having an issue with chocolatey not working (https://github.com/marticliment/UniGetUI/issues/3033) that for some of my packages, I may have conflicting or multiple installs. Here are some examples:

![image](https://github.com/user-attachments/assets/ca126efd-a023-44f4-93e0-b3ffd13ba45e)
![image](https://github.com/user-attachments/assets/b09b84dc-1b65-4fc0-8cc5-56ff9cb8267a)

![image](https://github.com/user-attachments/assets/2f12e039-60ec-4f70-915d-191ec5200933)

![image](https://github.com/user-attachments/assets/f66e3ee1-2608-45ec-96a2-24be3b7502e4)
![image](https://github.com/user-attachments/assets/469d7f83-f1ff-468b-b9d6-ebd1e5346693)

![image](https://github.com/user-attachments/assets/4d049215-5417-406b-9a62-a73462f0d0a5)
![image](https://github.com/user-attachments/assets/0d153291-8d64-42ab-8010-79444873206f)

---

How do you all manage multiple package managers, or have you transitioned to only using one package manager unless you can't find a package on a particular place? I guess what I'm trying to ask is what would be the appropriate approach towards this? Since I'm trying to transition to WinGet, is it better to uninstall the apps from chocolatey?

**Comment by hueldoeu** [comment]
is there a good reason to install chocolatey? i am satisfied with winget (and Pip. in the beginning i was confused that python packages i have never heard of needs to be updated as unigetUI notifies me).
![image](https://github.com/user-attachments/assets/45ff19f3-70f9-4e77-881c-6fb88cff9b77)


i don't want to install extra stuff

**Reply by marticliment** [reply]
No if you don't need it.
However, you might find some packages to work better with Chocolatey rather that WinGet, and the opposite

**Comment by mdgreenjr** [comment]
I don't see the question being asked - but what would be a great feature I think is to make it so that you can select let's say your 'primary' source for certain applications. As everyone has noticed, chocolatey sometimes lags behind winget a little bit but uniget will want to 'update' to the latest version and then we end up with 2 versions - an older choco one and the newer winget one.
